multiregion/docs/multiregion.rst

```diff
@@ -447,11 +534,87 @@ There is a script you can run to collect the JMX metrics from the command line,
       multi-region-sync: 4
       multi-region-async: 4
       multi-region-default: 4
+      under-min-isr-promotion: 4
+      under-replicated-promotion: 4
+      leader-is-observer-promotion: 4
 
 
 Failover and Failback
 ---------------------
 
+Degrade Region
```

Comment:
We should consider promoting (pun intended) `Degrade Region` into its own section, on the same level as `Failover and Failback`.  

1. Degrade is not a true failover
2. Degrade has merits in its own right

Reply:
good point, I've moved it up